src/constants/wallet.ts

```diff
@@ -56,20 +90,12 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
   },
   COINBASE_LINK: {
     name: 'Open in Coinbase Wallet',
+    wallet: Wallet.COINBASE_WALLET,
     iconURL: COINBASE_ICON_URL,
     description: 'Open in Coinbase Wallet app.',
     href: 'https://go.cb-w.com/mtUDhEZPy1',
     color: '#315CF5',
     mobile: true,
     mobileOnly: true,
   },
-  FORTMATIC: {
```

Comment:
Why are you deleting these wallets?

Reply:
commented above, but waiting on fortmatic to implement some EIPs